test/ttmlir/Silicon/TTNN/ttir_to_ttnn_pipeline.mlir

```diff
@@ -1,9 +1,9 @@
-// RUN: ttmlir-opt --ttir-to-ttnn-backend-pipeline %s  > %t.mlir
+// RUN: ttmlir-opt --ttir-load-system-desc --ttir-to-ttnn-backend-pipeline %s  > %t.mlir
```

Comment:
Why is this pass not part of the pipeline, especially if you need to call it every time?

Reply:
Yeah I agree, we can make it part of the pipeline.  It should be OK for the pass to run more than once in case you also want to manually specify.  @tapspatel, can you create a PR that adds this to the pipeline?

Reply:
yeah sorry about that, I added it here: https://github.com/tenstorrent/tt-mlir/pull/302

Reply:
https://github.com/tenstorrent/tt-mlir/pull/302